pages/osx/yabai.md

```diff
@@ -0,0 +1,24 @@
+# yabai
+
+> A tiling window manager for macOS based on binary space partitioning.
+> More information: <https://github.com/koekeishiya/yabai>.
+
+- Set the layout to bsp:
+
+`yabai -m config layout bsp`
+
+- Set the window gap:
```

Comment:
```suggestion
- Set the window gap to 10:
```

Maybe even `10 pixels` (?) if that's the correct unit of measure here.

Reply:
I've updated it to `to 10pt` instead. Based on the looks it's definitely pixels but on Yabai's documentation they didn't mention it as pixels, they did mention it as 10pt, though.

I've also updated the other lines based on your comments. Thanks!